web/components/src/Heading/Heading.tsx

```diff
@@ -27,10 +27,15 @@ const StyledHeading = styled(Typography)<StyledHeadingProps>`
   .inverted-background & {
     color: var(--inverted-text);
   }
+
+  strong,
+  b {
+    font-weight: var(--fontWeight-medium);
```

Comment:
why not bold?

Reply:
Nick's sketches use font-weight : 500 for bold , but in our code font-weight for medium is 500 and bold is 600

Reply:
won't this change already approved headers?  can you verify with Nick if this is correct, i would think that his sketch is not accurate